src/app/accounts/[id]/page.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import { useParams, useRouter } from "next/navigation";
+import { useAppStore } from "@/app/store/store";
+import { useEffect, useState } from "react";
+import Button from "@/app/components/form/Button";
+import Input from "@/app/components/form/Input";
+import styles from "./styles";
+import paths from "@/app/paths";
+
+export default function Home() {
```

Comment:
@kaiomagalhaes it seems that this is an account page, I'd suggest having a different structure

```
accounts/
[id].tsx (this could be the user profile page)
new.tsx
styles.ts
```

we could even rename this function
```suggestion
export default function Account() {
```